src/pages/tutorial/angular/step-2.mdx

```diff
@@ -0,0 +1,718 @@
+---
+title: 2. Building pages
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### Now that we have a base Angular app, it's time to build a few static pages. In this step, we'll become comfortable with the Carbon UI Shell, grid and various Carbon components.
+
+<AnchorLinks>
+
+<AnchorLink>Fork, clone and branch</AnchorLink>
+<AnchorLink>Add grid</AnchorLink>
+<AnchorLink>Add landing page grid</AnchorLink>
+<AnchorLink>Build landing page</AnchorLink>
+<AnchorLink>Style landing page</AnchorLink>
+<AnchorLink>Add repo page grid</AnchorLink>
+<AnchorLink>Build repo page</AnchorLink>
+<AnchorLink>Style repo page</AnchorLink>
+<AnchorLink>Submit pull request</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+A [preview](https://angular-step-3-carbon-tutorial.netlify.com) of what you'll build:
```

Comment:
React left, Angular right:

![image](https://user-images.githubusercontent.com/1691245/67426267-58353d00-f59f-11e9-8115-b45628b8ee16.png)

* Left/right edge of the data table does not align with UI shell (too much left/right whitespace between edges of viewport and the data table)
* Clicking "IBM Carbon Tutorial Angular" does not take you home

Reply:
The extra whitespace issue confused the heck out of me ... as it turns out the React tutorial is using a non-standard `bx--grid--no-gutter` class to reduce the `bx--grid` margins down to `1 rem` (??) ... After literally hours of searching I found this PR: https://github.com/carbon-design-system/carbon/pull/3435 which was closed because reasons. I searched via github and found no matches on the tutorial site for `no-gutter`

@mattrosno what's the path forward here? Should we stick with what the carbon grid provides (2rem padding) or add an override?

Reply:
So I went with adding the override ... This also fixed some of the bugs on the homepage